dist/api_err.py

```diff
@@ -109,9 +119,32 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
             if 'undoc' in err.flags:
                 tfile.write('/*! @endcond */\n')
         tfile.write('/*\n')
+    return skip
+
+# Update the #defines in the wiredtiger.in file.
+tmp_file = '__tmp_api_err' + str(os.getpid())
+tfile = open(tmp_file, 'w')
+skip = 0
+for line in open('../src/include/wiredtiger.in', 'r'):
+    if not skip:
+        tfile.write(line)
+    skip = check_write_errors(skip, tfile, 'Error', errors)
+    skip = check_write_errors(skip, tfile, 'Sub-level error', sub_errors)
```

Comment:
Do we need to give `skip` to those functions?

Reply:
I think we do, because if neither of the cases in the if else statement occur, then we need to return the current state of skip. This is because, when we first encounter the start of the error section, we need to continually skip lines until we reach the end of the error section. So we need to maintain the value of skip while going through the error section